src/Webroot.php

```diff
@@ -147,16 +147,20 @@ public function putassets($sspak) {
 		$assetsOldPath = $assetsPath . '.old';
 		$assetsParentArg = escapeshellarg(dirname($assetsPath));
 
+		// Remove assets.old
+		if (file_exists($assetsOldPath)) {
+			$this->execSudo("rm -rf {$assetsOldPath}");
+		}
+
 		// Move existing assets to assets.old
-		$this->exec("if [ -d {$assetsPath} ]; then mv {$assetsPath} {$assetsOldPath}; fi");
+		if (file_exists($assetsPath)) {
+			$this->execSudo("mv {$assetsPath} {$assetsOldPath}");
+		}
 
 		// Extract assets
 		$stream = $sspak->readStreamForFile('assets.tar.gz');
-		$this->exec("tar xzf - -C {$assetsParentArg}", array('inputStream' => $stream));
+		$this->execSudo("tar xzf - -C {$assetsParentArg}", array('inputStream' => $stream));
 		fclose($stream);
-
-		// Remove assets.old
-		$this->exec("if [ -d {$assetsOldPath} ]; then rm -rf {$assetsOldPath}; fi");
```

Comment:
This is also outside the scope of the PR - I'd include it in the above one, with your reasoning. I imagine it's something like "keeping a backup is better then saving space", but that's a different conversation 😄

Reply:
I must've misinterpreted what assets.old was for when I first was using sspak... Yea i'll put these back the other way, no reasoning to the reorder